examples/tutorial/00_Setup.ipynb

```diff
@@ -54,10 +54,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pandas as pd, datashader as ds, dask, bokeh, holoviews as hv  # noqa\n",
+    "import datashader as ds, bokeh, holoviews as hv  # noqa\n",
     "from distutils.version import LooseVersion\n",
     "\n",
-    "min_versions = dict(pd='0.24.0', ds='0.7.0', dask='1.2.0', bokeh='1.2.0', hv='1.12.3')\n",
+    "min_versions = dict(ds='0.11.0', bokeh='2.0.1', hv='1.13.2')\n",
```

Comment:
Why no longer check the dask version?

Reply:
Because we no longer have any real requirements against them. Last year the pandas 1.0 transition was going on so dask hadn't yet been updated for compatibility. At this point all that has settled down.